src/main/java/teammates/ui/controller/InstructorFeedbackPreviewAsInstructorAction.java

```diff
@@ -32,22 +32,24 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         InstructorAttributes previewInstructor = logic.getInstructorForEmail(courseId, previewInstructorEmail);
         
         if (previewInstructor == null) {
-            throw new EntityDoesNotExistException("Instructor Email "
-                    + previewInstructorEmail + " does not exist in " + courseId
-                    + ".");
+            throw new EntityDoesNotExistException("Instructor Email " +
+                      previewInstructorEmail + " does not exist in " + courseId +
+                      ".");
         }
         
         FeedbackSubmissionEditPageData data = new FeedbackSubmissionEditPageData(account, student);
         
         data.bundle = logic.getFeedbackSessionQuestionsBundleForInstructor(
-                feedbackSessionName, courseId, previewInstructor.email);
+                feedbackSessionName, 
+                courseId, 
+                previewInstructor.email);
         
         // the following condition is not tested as typically the GateKeeper above handles
```

Comment:
Was talking about these areas, comments tell us why it is not tested, otherwise the action files are 100% tested